private/solr-conf/synonyms_und.txt

```diff
@@ -11,35 +11,8 @@
 
 phone,telephone
 tv,television
-911 call,911 calls,dispatch call,dispatch calls,emergency call,emergency calls
-address,addresses,location,locations
 adu,accessory dwelling unit
-appeal,appeals,petition,petitions
-building permit,building permits,construction permit,construction permits
-business license,business registration,business permit
-business search=>business lookup
-certificate of compliance,certificate of conformance,certificate of conformity
-classes,lessons,instructions,training
-directory,catalogue,roster
-employee,staff,personnel
 garbage,rubbish,waste,trash,debris,litter
-gym,gymns,gymnasium,gynmnasiums,health club,health clubs,fitness club,fitness clubs,fitness center,fitness centers
-internship,internsips,apprenticeship,apprenticeships,traineeship,traineeships,work experience
-job,jobs,career,careers,employment
-job opening,job openings,job posting,job postings,job listing,job listings
-non emergency,non urgent,elective
-parking,car park
-pay bill,pay invoice,pay account,pay notice,pay charge,pay statement
-pay ticket,pay fine,settle ticket
-permit,permits,license,licenses
-police report,police reports,incident report,incident reports
-property tax,property taxes,real estate tax,real estate taxes
-schedule,agenda,plan,program
-short term rental,short term rentals,temporary housing,transitional housing
-summer camp,summer camps,summer activity,summer activities
-swimming,aquatic
-title 33,zoning code
```

Comment:
I'd recommend keeping this one too.

Reply:
The comment is showing me 4 lines, but I'm going to assume it is the last one that you are referring to with this comment. 

I would completely agree if the title of the page that should be the top result for either of those search terms is literally titled "Title 33 Planning and Zoning". I think it will make it to number one without the boost.